Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/DiagnosticsTab/DiagnosticsStatus.tsx

```diff
@@ -29,6 +30,10 @@ export const DiagnosticsStatus = () => {
       data-testid="diagnostics-tab-header--collapsed"
     >
       <DefaultButton
+        ariaLabel={formatMessage('{ errors } errors, { warnings } warnings', {
```

Comment:
Does it need pluralizing here? 1 error 1 warning?

Reply:
Good catch. I'll fix that too.